manage_breast_screening/clinics/jinja2/show.jinja

```diff
@@ -13,35 +14,90 @@
 
 {% block page_content %}
-    <h1 class="nhsuk-heading-l">
-        <span class="nhsuk-caption-l">{{ presented_clinic.setting_name }}</span>
-        {{ presented_clinic.risk_type }} screening clinic
-
-        {{ tag({
-            "html": presented_clinic.state.text | no_wrap,
-            "classes": presented_clinic.state.classes
-        })}}
-    </h1>
-    <p>{{ presented_clinic.time_range }} - {{ presented_clinic.starts_at }}</p>
-
-    {% set ns = namespace() %}
-    {% set ns.secondaryNavItems = [] %}
-    {% for item in [
-      { "filter": 'remaining', "label": 'Remaining' },
-      { "filter": 'checked_in', "label": 'Checked in' },
-      { "filter": 'complete', "label": 'Complete' },
-      { "filter": 'all', "label": 'All' }
-    ] %}
-    {% set href %}/clinics/{{ presented_clinic.id }}/{{ item.filter }}{% endset %}
-    {% set ns.secondaryNavItems = ns.secondaryNavItems + [{
-      "text": (item.label + " " + appCount(presented_appointment_list.counts_by_filter[item.filter])) | safe,
-      "href": href | trim,
-      "current": true if item.filter == presented_appointment_list.filter
-    }] %}
-    {% endfor %}
-
-    {{ app_secondary_navigation({
-      "visuallyHiddenTitle": "Secondary menu",
-      "items": ns.secondaryNavItems
-    }) }}
+  <h1 class="nhsuk-heading-l">
+      <span class="nhsuk-caption-l">{{ presented_clinic.setting_name }}</span>
+      {{ presented_clinic.risk_type }} screening clinic
+
+      {{ tag({
+          "html": presented_clinic.state.text | no_wrap,
+          "classes": presented_clinic.state.classes
+      })}}
+  </h1>
+  <p>{{ presented_clinic.time_range }} - {{ presented_clinic.starts_at }}</p>
+
+  {% set ns = namespace() %}
+  {% set ns.secondaryNavItems = [] %}
+  {% for item in [
+    { "filter": 'remaining', "label": 'Remaining' },
+    { "filter": 'checked_in', "label": 'Checked in' },
+    { "filter": 'complete', "label": 'Complete' },
+    { "filter": 'all', "label": 'All' }
+  ] %}
+  {% set href %}/clinics/{{ presented_clinic.id }}/{{ item.filter }}{% endset %}
+  {% set ns.secondaryNavItems = ns.secondaryNavItems + [{
+    "text": (item.label + " " + appCount(presented_appointment_list.counts_by_filter[item.filter])) | safe,
+    "href": href | trim,
+    "current": true if item.filter == presented_appointment_list.filter
+  }] %}
+  {% endfor %}
+
+  {{ app_secondary_navigation({
+    "visuallyHiddenTitle": "Secondary menu",
+    "items": ns.secondaryNavItems
+  }) }}
+
+  {% set table_rows = [] %}
+  {% for presented_appointment in presented_appointment_list.appointments %}
+    {% set details_html %}
+      <p class="nhsuk-u-margin-bottom-1">
+        <a href="{{ url('mammograms:start_screening', kwargs={"id": presented_appointment.id}) }}" class="nhsuk-link">
+          {{ presented_appointment.participant.full_name }}
+        </a>
+      </p>
+      <p class="app-text-grey nhsuk-u-margin-bottom-0">
+        NHS:  {{ presented_appointment.participant.nhs_number }}
```

Comment:
Minor, but I think @edwardhorsford  mentioned before we might want to extract this little paragraph to a partial due to it appearing in multiple places (or was that something else?) I'm happy to wait until we have more of the templates written and then refactor though.

I wonder also if there might be a need to have a little button to copy these numbers to clipboard for ease of working with other systems?